src/lib/viewers/BaseViewer.js

```diff
@@ -1081,6 +1081,30 @@ class BaseViewer extends EventEmitter {
         this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
     }
 
+    /**
+     * Handler for annotation mode change
+     * 1. Set annotationsControls mode
+     * 2. If discoverability FF is on, add mode classes to container
+     *
+     * @param {AnnotationMode} mode Next annotation mode
+     */
+    processAnnotationModeChange = mode => {
+        if (this.annotationControls) {
```

Comment:
It seemed like previously, if `this.annotationControls` didn't exist, no further processing would occur, but this now decouples the presence of `this.annotationControls` from managing the classes on `this.containerEl`. Is that desired?

Reply:
No, we shouldn't decouple these two. If `annotationsControls` doesn't exist, there is no meaning managing the classes on `this.container`. Will add a negative guard clause at the top of this function.